tensorflow/python/estimator/inputs/numpy_io.py

```diff
@@ -132,6 +132,8 @@ def input_fn():
     features = dict(zip(ordered_dict_x.keys(), features))
     if y is not None:
       target = features.pop(unique_target_key)
+      # undo modification to dictionary
+      del x[unique_target_key]
```

Comment:
Ok, this PR is then obsolete, there was no dictionary copy at the time, and the modification was basically done directly on the `x` instead of the `ordered_dict_x`. I failed to check the code after rebasing, my modification was based on this [version](https://github.com/tensorflow/tensorflow/blob/r1.1/tensorflow/python/estimator/inputs/numpy_io.py#L102)